porting/index.md

```diff
@@ -50,6 +55,8 @@ repo init -u https://github.com/Halium/android -b halium-5.1
 
 #### Step 3: Download the source code
 
+*Note:* the repo tool takes some time to download the sources. You need a little patience here. Also it could make sense to first read the next part about the Android tree configuration, since there you need another repo sync call to download everything.
```

Comment:
If you first download the parts from halium manifest.xml, then add your local_manifest entries, then you have to repo sync again...